.github/workflows/continuous_integration.yml

```
@@ -0,0 +1,137 @@
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          STANDARD: './phpcs.xml.dist'
+          EXTENSIONS: 'php,module,inc,install,profile,engine,theme,css'
+          REPORT: 'checkstyle'
+          IGNORE: '*.md,*.min.css,styles.css,wysiwyg.css,proofing.css,user_guides.css,whats_new.css,*/node_modules/*,*/simplesaml*/*,graphiql.css'
+          DIRECTORIES: 'docroot/modules/custom docroot/themes/custom tests'
+
+  # Analyze the codebase for programming errors with PHPStan.
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/post-checkout
+      - name: Run PHPStan and ReviewDog
+        run: |
+          export REVIEWDOG_GITHUB_API_TOKEN="${GITHUB_TOKEN}"
+          bin/phpstan analyze \
+            --no-progress \
+            --no-interaction \
+            --error-format=raw \
+            --memory-limit=1G \
+            -q \
+            ${DIRECTORIES} \
+            | reviewdog \
+              -name="PHPStan" \
+              -f=phpstan \
+              -reporter=github-pr-review \
+              -diff='git diff'
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          STANDARD: './phpcs.xml.dist'
+          EXTENSIONS: 'php,module,inc,install,profile,engine,theme,css'
+          REPORT: 'checkstyle'
+          IGNORE: '*.md,*.min.css,styles.css,wysiwyg.css,proofing.css,user_guides.css,whats_new.css,*/node_modules/*,*/simplesaml*/*,graphiql.css'
+          DIRECTORIES: 'docroot/modules/custom docroot/themes/custom tests'
+
+  # Run unit tests against codebase with PHPUnit.
+  phpunit:
+    name: PHPUnit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/post-checkout
+      - name: Run PHPUnit (Unit Tests only)
+        run: bin/phpunit --group unit --exclude-group disabled tests/phpunit
+
+  # Check styles of modules and themes with Stylelint.
+  stylelint:
```

There were two workflows, `stylelint-modules` and `stylelint-themes`, but both were called `stylelint` and thus both were responsible for the same status check.  Whichever workflow executed last would set the status check, so this might have been masking errors.